SpatialGDK/Source/SpatialGDK/Private/LoadBalancing/GridBasedLBStrategy.cpp

```diff
@@ -15,12 +15,13 @@ UGridBasedLBStrategy::UGridBasedLBStrategy()
 	, Cols(1)
 	, WorldWidth(10000.f)
 	, WorldHeight(10000.f)
+	, InterestBorder(0.f)
 {
 }
 
-void UGridBasedLBStrategy::Init(const USpatialNetDriver* InNetDriver)
+void UGridBasedLBStrategy::Init()
```

Comment:
Soooo, can we just the init functionality to the constructor and delete Init()? 😄

Reply:
probably

Reply:
results are in- tentative yes. removed some inits, but moved one to the constructor.

this is starting to look like one of those innocent drive bys that make my PRs very controversial and prone to not getting merged

Reply:
now only one worker is getting delegated authority over anything

Reply:
does doing things in the constructor mean that it doesn't get the overwritten values?

Reply:
reverted because pretty sure it meant it would only ever use default values in the init